docs/md/docs/2020-11-02-turn-the-senior-around.md

```diff
@@ -0,0 +1,68 @@
+# Turn The Senior Around
+
+As engineering managers we take many considerations into account, two of the main ones are:
+- Delivering on time
+- Make our people grow
```

Comment:
The use of **_Make_** sounds like growing their team members is the Engineering Manager's responsibility. IMHO a rephrase of that sentence to "Support your team members professional growth and development." shits that a bit towards the team members